### PR TITLE
Minor clarifications and formatting tweaks

### LINKED ISSUE #7788: Minor tweaks
Update: I see that I can create a pull request for you. I may do that instead of complaining here. :-)

I'd like to make some minor tweaks such as:
* Using "i.e." where it should be "e.g."
* Removing extraneous spaces and punctuation such as the extra space and comma in this: "in the case of the LocBaml Tool, ( see Localize an Application),".
* Fix broken code formatting on the last code block in the article.

I might also do something like add some extra screenshots to show what the code blocks (e.g., the one about the vertical bar separator) do and clarify where the `&lt;UICulture&gt;` tags go in the .csproj file.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4baeeb7d-bba4-c8c5-6f54-97dc68a59aef
* Version Independent ID: d248598d-5657-1a8c-3f91-b37384243eff
* Content: [WPF Globalization and Localization Overview](https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/wpf-globalization-and-localization-overview)
* Content Source: [docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md](https://github.com/dotnet/docs/blob/master/docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md)
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by mlimber**
## Minor clarifications and formatting tweaks

I applied some minor fixes such as:
* Clarify where the `<UICulture>` element goes in the .csproj file.
* Using "e.g." (for example) instead of "i.e." (that is).
* Removing extraneous spaces and punctuation such as the extra space and comma in this: "in the case of the LocBaml Tool, ( see Localize an Application),".
* Fix broken code formatting on the last code block in the article. (Did not convert to external file to provide minimum changeset.)

Fixes #7788 (reported by me before I realized I could edit directly).

### CODE REVIEW

File: docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md [hunk 1]
```diff
@@ -191,7 +193,7 @@ When you limit your product's availability to only one language, you limit your
...
   
- On German [!INCLUDE[TLA#tla_mswin](../../../../includes/tlasharptla-mswin-md.md)], if this resources.dll is placed in a de-DE folder next to the main assembly, this resource will automatically load instead of the one in the en-US folder. If you do not have a German version of [!INCLUDE[TLA#tla_mswin](../../../../includes/tlasharptla-mswin-md.md)] to test this, set the culture to whatever culture of [!INCLUDE[TLA#tla_mswin](../../../../includes/tlasharptla-mswin-md.md)] you are using (i.e. en-US), and replace the original resources.dll.  
+ On German [!INCLUDE[TLA#tla_mswin](../../../../includes/tlasharptla-mswin-md.md)], if this resources.dll is placed in a de-DE folder next to the main assembly, this resource will automatically load instead of the one in the en-US folder. If you do not have a German version of [!INCLUDE[TLA#tla_mswin](../../../../includes/tlasharptla-mswin-md.md)] to test this, set the culture to whatever culture of [!INCLUDE[TLA#tla_mswin](../../../../includes/tlasharptla-mswin-md.md)] you are using (e.g., `en-US), and replace the original resources.dll.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md]: we actually shouldn't use neither i.e. or e.g. in our docs and actually spell it out for example.
also, make sure that the other missing backtick is added.

File: docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md [hunk 2]
```diff
@@ -269,26 +271,20 @@ When you limit your product's availability to only one language, you limit your
...
+ If you specify a font that does not support a given codepoint range, [!INCLUDE[TLA2#tla_winclient](../../../../includes/tla2sharptla-winclient-md.md)] will automatically fallback to one that does by using the Global User Interface.compositefont that is located in your Windows\Fonts directory. Composite fonts work just as any other font and can be used explicitly by setting an element’s FontFamily (i.e., `FontFamily="Global User Interface"`). You can specify your own font fallback preference by creating your own composite font and specifying what font to use for specific codepoint ranges and languages.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/wpf-globalization-and-localization-overview.md]: should this one be for example as well?

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you for contributing to our docs @mlimber. Changes look great but left a few comments for a couple of things that should be fixed before we could merge. Thanks!

**Comment by mlimber** [comment]
I updated my PR. Let me know if you need anything else from me! Thanks.

**Review by mairaw — APPROVED** [review]
This looks great @mlimber. I've just fixed some of escaping of the asterisks before we can merge.

**Pull request merged**